lib/runner.js

```diff
@@ -20,6 +23,7 @@ var chainableMethods = {
 		before: {type: 'before'},
 		after: {type: 'after'},
 		skip: {skipped: true},
+		todo: {skipped: true, todo: true},
```

Comment:
It might be better to not set `skipped: true` here. That would require further clauses where `skipped` is used to determine whether the test is skipped, but it also removes exceptions where you have to check for `!todo`.

@vdemedes thoughts?